pkg/utils/utils.go

```diff
@@ -238,3 +239,18 @@ func GetUIDLastSection(uid types.UID) string {
 	}
 	return uidLastSection
 }
+
+func CompareFiles(filePath1, filePath2 string) (bool, error) {
```

Comment:
nit.. mixed function params is generally not recommended. Let's mention the type against filePath1 too.

Reply:
Done